packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_single.dart

```diff
@@ -138,95 +139,79 @@ class _ShallowSizeColumn extends ColumnData<SingleClassStats> {
 }
 
 class _RetainedSizeColumn extends ColumnData<SingleClassStats> {
-  _RetainedSizeColumn(this.totalSize)
+  _RetainedSizeColumn(this.classedData)
       : super(
           'Retained Dart Size',
           titleTooltip: SizeType.retained.description,
           fixedWidthPx: scaleByFontFactor(140.0),
           alignment: ColumnAlignment.right,
         );
 
-  final int totalSize;
+  final ClassesTableSingleData classedData;
 
   @override
-  int getValue(SingleClassStats classStats) => classStats.objects.retainedSize;
+  int getValue(SingleClassStats dataObject) => dataObject.objects.retainedSize;
 
   @override
   bool get numeric => true;
 
   @override
-  String getDisplayValue(SingleClassStats classStats) {
-    final value = getValue(classStats);
+  String getDisplayValue(SingleClassStats dataObject) {
+    final value = getValue(dataObject);
 
     final bytes = prettyPrintRetainedSize(value)!;
 
-    final percents = '${(value * 100 / totalSize).round()}%';
+    final percents = '${(value * 100 / classedData.totalHeapSize()).round()}%';
```

Comment:
use the `percent2` helper from utils.dart. You can change change this helper method to take a `fractionDigits` parameter that defaults to 2 so that it can be modified if you want 0 digits here.